dotnetv3/EC2/Scenarios/EC2_Basics/EC2Basics.cs

```diff
@@ -0,0 +1,301 @@
+        Console.WriteLine($"\tssh -i {tempFileName} ec2-user@{instance.PublicIpAddress}");
+
+        uiMethods.PressEnter();
+
+        Console.WriteLine("Now we'll stop the instance and then start it again to see what's changed.");
+
+        await ec2Methods.StopInstances(instanceId);
+        var hasStopped = false;
+        do
+        {
+            hasStopped = await ec2Methods.WaitForInstanceState(instanceId, InstanceStateName.Stopped);
+        } while (!hasStopped);
+
+        Console.WriteLine("\nThe instance has stopped.");
+
+        Console.WriteLine("Now let's start it up again.");
+        await ec2Methods.StartInstances(instanceId);
+        Console.Write("Waiting for instance to start. ");
+
+        isRunning = false;
+        do
+        {
+            isRunning = await ec2Methods.WaitForInstanceState(instanceId, InstanceStateName.Running);
+        } while (!isRunning);
+
+        Console.WriteLine("\nLet's see what changed.");
+
+        instance = await ec2Methods.DescribeInstance(instanceId);
+        uiMethods.DisplayTitle("New Instance Information");
+        ec2Methods.DisplayInstanceInformation(instance);
+
+        Console.WriteLine("\nNotice the change in the SSH information:");
+        Console.WriteLine($"\tssh -i {tempFileName} ec2-user@{instance.PublicIpAddress}");
+
+        uiMethods.PressEnter();
+
+        Console.WriteLine("Now we will stop the instance again. Then we will create and associate an");
+        Console.WriteLine("Elastic IP address to use with our instance.");
+
+        await ec2Methods.StopInstances(instanceId);
+        hasStopped = false;
+        do
+        {
+            hasStopped = await ec2Methods.WaitForInstanceState(instanceId, InstanceStateName.Stopped);
+        } while (!hasStopped);
+
+        Console.WriteLine("\nThe instance has stopped.");
+        uiMethods.PressEnter();
+
+        uiMethods.DisplayTitle("Allocate Elastic IP Address");
+        Console.WriteLine("You can allocate an Elastic IP address and associate it with your instance\nto keep a consistent IP address even when your instance restarts.");
+        var allocationId = await ec2Methods.AllocateAddress();
+        Console.WriteLine("Now we will associate the Elastic IP address with our instance.");
+        var associationId = await ec2Methods.AssociateAddress(allocationId, instanceId);
+
+        // Start the instance again.
+        Console.WriteLine("Now let's start the instance again.");
+        await ec2Methods.StartInstances(instanceId);
+        Console.Write("Waiting for instance to start. ");
+
+        isRunning = false;
+        do
+        {
+            isRunning = await ec2Methods.WaitForInstanceState(instanceId, InstanceStateName.Running);
+        } while (!isRunning);
+
+        Console.WriteLine("\nLet's see what changed.");
+
+        instance = await ec2Methods.DescribeInstance(instanceId);
+        uiMethods.DisplayTitle("Instance Information");
```

Comment:
lowercase "information" (here and elsewhere, use sentence casing in titles, capitalizing first word and lowercasing others)

Reply:
changed